docs/sources/dashboards/dashboard-public/index.md

```diff
@@ -221,7 +222,7 @@ guaranteed because plugin developers can override this functionality. The follow
     </td>
     <td>
       <ul>
-        <li>Dynatrace</li>
+        <li>DynamoDB</li>
```

Comment:
in the supported escalation issue, it says that DynamoDB is confirmed as unsupported, not as unconfirmed. I think we should, therefore, put it in `Unsupported` list

Reply:
Whoops. I reviewed the other PR, but didn't look at the original escalation. For some reason, the person who wrote the escalation thought that confirmed unsupported ones should be put under Unconfirmed. No idea why. I'll revert the other PR and fix that on main.

Reply:
should we also include `DynamoDB` in `shared-dashboards` doc?

Reply:
That's being done on PR https://github.com/grafana/grafana/pull/99120